diff --git a/src/resources/lib/modules/updates.py b/src/resources/lib/modules/updates.py
index b13ad8a..5139750 100644
--- a/src/resources/lib/modules/updates.py
+++ b/src/resources/lib/modules/updates.py
@@ -46,7 +46,7 @@ class updates:
                     'settings': {
                         'AutoUpdate': {
                             'name': 32014,
-                            'value': 'auto',
+                            'value': 'manual',
                             'action': 'set_auto_update',
                             'type': 'multivalue',
                             'values': ['auto', 'manual'],
@@ -63,7 +63,7 @@ class updates:
                             },
                         'UpdateNotify': {
                             'name': 32365,
-                            'value': '1',
+                            'value': '0',
                             'action': 'set_value',
                             'type': 'bool',
                             'InfoText': 715,
